packages/plugin/webpack/test/AssetRelocatorPatch_spec.ts

```diff
@@ -145,14 +145,13 @@ describe('AssetRelocatorPatch', () => {
     });
 
     it('builds preload', async () => {
-      const entryPoint = config.renderer.entryPoints[0] as WebpackPluginEntryPointLocalWindow;
-      const preloadConfig = await generator.getPreloadConfigForEntryPoint(entryPoint);
-      await asyncWebpack(preloadConfig);
+      const preloadConfig = await generator.getRendererConfig(config.renderer.entryPoints);
+      await asyncWebpack(preloadConfig[1]);
 
       await expectOutputFileToHaveTheCorrectNativeModulePath({
-        outDir: path.join(rendererOut, 'main_window'),
+        outDir: rendererOut,
         jsPath: path.join(rendererOut, 'main_window/preload.js'),
-        nativeModulesString: '__webpack_require__.ab = __dirname + "/native_modules/"',
+        nativeModulesString: `__webpack_require__.ab = ${JSON.stringify(rendererOut)} + "/native_modules/"`,
```

Comment:
This seems wrong, doesn't that mean we have a hardcoded path (from the build machine) being injected into the final output 🤔

Reply:
If I am understanding this code correctly, we do in fact have a hardcoded path being injected in XD
<img width="711" alt="image" src="https://user-images.githubusercontent.com/33054982/219211357-748e129b-34fa-4035-89be-4eeaae157214.png">